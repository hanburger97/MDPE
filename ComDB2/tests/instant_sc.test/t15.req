select a from t
select a from t order by i

insert into t(i, j) values(36, 36)
insert into t(i, j, a) values(37, 37, 37)

select a from t
select a from t order by i
