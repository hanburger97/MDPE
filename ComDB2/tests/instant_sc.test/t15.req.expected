(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(rows inserted=1)
(rows inserted=1)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=37)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=NULL)
(a=37)
